exonum/src/crypto/x25519.rs

```diff
@@ -88,7 +88,7 @@ pub fn scalarmult_base(sc: &SecretKey) -> PublicKey {
 /// Converts Ed25519 public key to Curve25519 public key.
 ///
 /// See: [`into_x25519_keypair()`][1]
-/// [1]: fn.into_x25519_public_key.html
+/// [1]: `fn.into_x25519_public_key.html`
```

Comment:
Is it needed indeed? It's just a link.

Reply:
It required by doc_markdown lint

Reply:
Then why some similar cases below aren't formatted this way?

Reply:
It is question to clippy ;)

Reply:
I guess it is the same situation like use_self